GitExtensions/Program.cs

```diff
@@ -54,27 +54,22 @@ private static void Main()
             }
 
             string[] args = Environment.GetCommandLineArgs();
-            FormSplash.ShowSplash();
-            //Store here SynchronizationContext.Current, because later sometimes it can be null
-            //see http://stackoverflow.com/questions/11621372/synchronizationcontext-current-is-null-in-continuation-on-the-main-ui-thread
-            GitUIExtensions.UISynchronizationContext = SynchronizationContext.Current;
-            AsyncLoader.DefaultContinuationTaskScheduler = TaskScheduler.FromCurrentSynchronizationContext();
-            Application.DoEvents();
+
+            //This form created for obtain UI synchronization context only
+            using (var empFormForSyncContext = new Form())
```

Comment:
can be simplified to `using (new Form())`

Reply:
Good catch. Thanks! Done.